### PR TITLE
975: Show more user information 

### LINKED ISSUE #975: Show more user information
As a user I want to see information about my user when i'm logged in such as username and role

## Acceptance Criteria:
 - show username
 - show role

### PR DESCRIPTION
**Pull request by f1sh1918**
added a user menu and show some user information

I had to make the popover controlled since lot of things not working properly for the `Popover` component like backdrop.
I think we have to update to the new blueprint version at some point :)

resolves #975

### CODE REVIEW

File: administration/src/bp-modules/UserMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+const UserMenu = ({ isOpen, setIsOpen, onSignOut }: UserMenuProps): ReactElement => {
+  const { role, email } = useContext(WhoAmIContext).me!
+  const navigate = useNavigate()
+  const signOutAndRedirect = () => {
+    onSignOut()
+    navigate('/')
+  }
+  const userMenuContent = (
+    <MenuContent onClick={() => setIsOpen(false)}>
+      <span className='bp5-tag bp5-large'>{`Rolle: ${roleToText(role)}`}</span>
```
Comment (sarahsporck) [inline administration/src/bp-modules/UserMenu.tsx:49]: Why not use `Tag` directly? In the current implementation this looks like a selected button. Don't like that.

Comment (f1sh1918) [reply administration/src/bp-modules/UserMenu.tsx:49]: For me it looks like a badge? :/ tag is deprecated in the api docs

Comment (f1sh1918) [reply administration/src/bp-modules/UserMenu.tsx:49]: what about this @sarahsporck 
image

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox

**Pull request merged**